### ISSUE TITLE
How do I run the unit tests?

### ISSUE THREAD

**Issue by macsj200** [wweiss/aws-serverless-tweet-event-source#13]
I try `yarn test` and it doesn't seem to run the tests. I also tried `yarn jest` and that fails due to:

```
    import { LoggerFactory } from '@codification/cutwater-logging';
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module
```

**Comment by wweiss** [comment]
Hi!  Sorry your running into issues.  In reality, I'm moving away from this type of solution and moving to CDK implementations - just cleaner.  However, I'll see if I can help.
I tried doing a clean clone and run and everything was fine here...
```
yarn run v1.22.1
$ gulp test --clean
[11:38:24] Using gulpfile ~\Projects\aws-serverless-tweet-event-source\gulpfile.js
[11:38:24] Starting 'test'...
[11:38:24] Starting subtask 'clean'...
[11:38:24] Finished subtask 'clean' after 19 ms
[11:38:24] Starting subtask 'tsLint'...
[11:38:26] Finished subtask 'tsLint' after 1.94 s
[11:38:26] Starting subtask 'tsc'...
[11:38:26] Starting subtask 'tsc-es6'...
[11:38:26] Starting subtask 'copy-static-assets'...
[11:38:26] Finished subtask 'copy-static-assets' after 58 ms
[11:38:46] Finished subtask 'tsc' after 20 s
[11:38:46] Finished subtask 'tsc-es6' after 20 s
[11:38:46] Finished 'test' after 22 s
[11:38:46] ==================[ Finished ]==================
[11:38:46] Project aws-serverless-tweet-event-source version:0.0.4
[11:38:46] Build tools version:0.21.0
[11:38:46] Node version:v14.16.0
[11:38:46] Total duration:22 s
Done in 65.08s.
```
This was after running `yarn install` on a clean clone, followed by `yarn test` on a Windows 10 machine.
More details on your setup might help me troubleshoot...

**Comment by wweiss** [comment]
Should be solved by 3a46b75.  Pull and try running `yarn test` again.

**Comment by wweiss** [comment]
Let me know if this is still an issue.
